workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Playground
+{
+    public class EntityGameObjectCreator
+    {
+        private readonly World world;
+        private readonly Dictionary<string, GameObject> cachedPrefabs;
+
+        public EntityGameObjectCreator(World world, Dictionary<string, GameObject> initialCachedPrefabs)
+        {
+            this.world = world;
+            cachedPrefabs = initialCachedPrefabs;
```

Comment:
Is this just here for testing?

Reply:
In which case can we handle the null case, or give a constructor that doesn't ask for an initial cache?

Reply:
Added null case.